sygnal/gcmpushkin.py

```diff
@@ -482,23 +493,15 @@ def __init__(self):
             mapping of registration ID to either its canonical registration ID,
             or `None` if there is no entry.
         """
-        return {reg_id: await self.get_canonical_id(reg_id) for reg_id in reg_ids}
```

Comment:
not saying it's a bad idea, but... is this actually part of supporting postgres or is it an unrelated change?

Reply:
(for clarity: I'm not saying you need to change it back now, but for future ref, assuming it is unrelated, I'd prefer it to be in a separate PR.)

Reply:
Yep, I got a bit eager with changing something that seemed like a performance problem. Luckily there are unit tests testing this path. In the future, I will open a separate PR :+1: